### DISCUSSION TITLE
AlDente 1.16

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#657] [General]
* Top Up now does not automatically change the charging percentage back when 100% is reached to allow trickle-charging The original percentage is still set back when the power cord is unplugged or another mode is applied
* Added the capability to use the Hardware Battery Percentage instead of the macOS value. The Hardware Battery Percentage is much more accurate, you can read more about this feature on our blog: https://apphousekitchen.com/feature-explanation-hardware-battery-percentage/
* Added Turkish language support
* Several other small changes and fixes
​
With Love and support for Ukraine 🇺🇦
A🏠K

<hr /><em>This discussion was created from the release <a href='https://github.com/davidwernhart/AlDente/releases/tag/1.16'>AlDente 1.16</a>.</em>

**Comment by kentzh1** [comment]
Great, hope it gets better
